### PR TITLE
Specs for opcode GAS (0x5A)

### PR DESCRIPTION
**Pull request by roynalnaruto**
Addresses #21

### CODE REVIEW

File: specs/opcode/51GAS.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# GAS opcode
```
Comment (DreamWuGit) [inline specs/opcode/51GAS.md:1]: file name should be specs/opcode/5AGAS.md ?

Comment (roynalnaruto) [reply specs/opcode/51GAS.md:1]: Oh, yes. I am not sure how I managed to mess that up. Good find!

**Review by lispc — APPROVED** [review]

**Review by DreamWuGit — APPROVED** [review]
LGTM

**Comment by icemelon** [comment]
Close and move to https://github.com/appliedzkp/zkevm-specs/pull/104

**Pull request closed without merging**